src/backend/storeManagers/storeManagerCommon/games.ts

```diff
@@ -252,8 +253,16 @@ export async function launchGame(
     executable = gameSettingsOverrides.targetExe
   }
 
-  if (browserUrl) {
-    return openNewBrowserGameWindow(browserUrl, gameInfo)
+  let { browserUrl } = gameInfo
+  if (platform === 'web') {
+    const webGameUrl =
+      gameInfo?.channels?.[channelName ?? ''].release_meta.platforms[platform]
```

Comment:
to handle `channelName` being falsy, I think it might be cleaner to do something like

```typescript
let webGameUrl;

if (gameInfo?.channels && channelName !== '') {
    webGameUrl = gameInfo.channels.[channelName].release_meta.platforms[platform]
}
```

Reply:
updated now